ComicLibrary.spec

```
@@ -0,0 +1,38 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+a = Analysis(
+    ['src\\my_project\\main.py'],
+    pathex=[],
```

RuboCop (1.88.2)

[warning] Useless assignment to variable - `pathex`.

(Lint/UselessAssignment)